homeassistant/components/qvr_pro/__init__.py

```diff
@@ -82,7 +87,7 @@ def setup(hass: HomeAssistant, config: ConfigType) -> bool:
 
     hass.data[DOMAIN] = {"channels": channels, "client": qvrpro}
 
-    load_platform(hass, CAMERA_DOMAIN, DOMAIN, {}, config)
+    load_platform(hass, Platform.CAMERA, DOMAIN, {}, config)
```

Comment:
Please keep this change on hold until #63571 is approved

Reply:
Care to share why @epenet?
As in, the current typing is already valid? Thus this is already valid?

Reply:
We have changed everywhere the `PLATFORMS` constants (or pretty much everywhere).
But for now there are 100s of places where `load_platform` and `async_load_platform` are still using a string or the imported domain. I therefore felt that since this PR was mixing up the two (some changes to PLATFORM constants, and some changes to load_platform method it was better to postpone the method changes.

Reply:
Do we consider that `async_setup_component` allows a `Platform` enum?
https://github.com/home-assistant/core/blob/b3c2ebd9ca7f1116bd1076d6cdb073e2699a2baa/homeassistant/helpers/discovery.py#L162

Reply:
I think that until we approve the change to the type hint, then we shouldn't use the Platform enum more.
Thus I created PRs #63571 / #63581 to get confirmation.

Reply:
Both PRs have been merge. I think it's safe to merge this one now too.